docs/example.md

```diff
@@ -19,11 +19,11 @@ We used the [spreadsheet template](spreadsheet.md) to prepare a series of releas
 
 This was exported as an Excel file, and converted into JSON using the [OCDS Convert, Validate and Explore tool](http://standard.open-contracting.org/validator/)
 
-The release were then compiled into a record, and are available to browse using the [open source OCDS Show framework](https://github.com/open-contracting/ocds-show/tree/ppp) which provides:
+The release were then compiled into a record, and are available to browse using the [open source OCDS Show framework](https://github.com/open-contracting/ocds-show/) which provides:
```

Comment:
I think the two links should be to OCDS Show for PPPs, right? https://github.com/open-contracting/ocds-show-ppp I prefer to hyperlink only the words "OCDS Show" and change "framework" to "tool" as it's not clear to me how it is a framework.

Should there be a link to the deployed app? https://open-contracting.github.io/ocds-show-ppp/

release -> releases

Reply:
> I think the two links should be to OCDS Show for PPPs, right? 

My understanding is that the OCDS Show for PPPs repo only exists so that there can be Github pages deployments of both versions.

The main OCDS show repo includes templates for the PPP schema (just not the theme and different title in the deployed version) and there is more comprehensive documentation in the readme, so I think it's best to link to that one.

Maybe we should add a link from the main OCDS-show readme to the OCDS-show for PPPs repo though?
> Should there be a link to the deployed app? https://open-contracting.github.io/ocds-show-ppp/

There is a link to the version deployed as part of the profile, with the example data loaded into it, under the next heading ('Explore the example'). Was there a particular reason to also provide a link to the (empty) Github pages version?

I've addressed the other points in https://github.com/open-contracting-extensions/public-private-partnerships/pull/209/commits/6bf3ebefdced2d3ddbb7e700846864414ec2bb9f

Reply:
(1) There is a link to OCDS Show from OCDS Show for PPPs, but feel free to suggest changes to make this more clear and the readme more helpful.

I think it's confusing for someone wanting to run a copy of OCDS Show for PPPs to be directed to OCDS Show, which by default doesn't use the PPP templates (they are in the repository, but not used by that app without changes). To use the PPP templates, the user at minimum needs to change index.html, specifically: change record_release.html to record_release_ppp.html, change release.html to release_ppp.html, and uncomment social witness code.

This is not obvious and is extra effort compared to using the repo that already has those changes made.

(2) Presently, the situation is:

1. Make changes to OCDS Show base templates
1. Copy those to OCDS Show for PPPs templates
1. Copy all changes to OCDS Show for PPPs
1. Copy all changes to OCDS for PPPs profile

This is a lot of steps, and different people at different times have forgotten that this is what is required, and so, for example:

1. David and Bibiana made changes to base templates, but …
1. … without copying to PPP templates. https://github.com/open-contracting/ocds-show/pull/30
1. Those same changes were not copied to OCDS Show for PPPs.
1. You made changes to the copy in the profile, which would be overwritten the next time we pulled in a copy. #178

I've made mistakes, too, and this is just a subset of the more times we have forgotten the process.

Clearly, this is overly complicated. Instead of a link like:

    ../_static/ocds-show/?load=../full_record_package.json

It can be:

    https://open-contracting.github.io/ocds-show-ppp/?load=https://standard.open-contracting.org/profiles/ppp/latest/en/_static/full_record_package.json

In which case we no longer need a local copy, eliminating the 4th step. To do that, we'd need to set CORS headers on the standard documentation server.

The 2nd step is the subject of https://github.com/open-contracting/ocds-show/issues/38 The 3rd step can perhaps be eliminated by having a common git submodule that both ocds-show and ocds-show-ppp use, or we can create an npm package that has only templates and they can declare it as a dependency. Anyway, those would be addressed elsewhere.

Reply:
If those changes are made, we'll also need to change the `update_ocds_show` make command. Basically: trim it down to just update `docs/examples` by `curl`'ing the two files from the ocds-show-ppp repository.

Reply:
Anyhow, we can do those further steps as part of https://github.com/open-contracting-extensions/public-private-partnerships/issues/211